### ISSUE TITLE
The replay text field still lacks clarity

### ISSUE THREAD

**Issue by machawk1** [oduwsdl/ipwb#415]
In #406 we improved the ipwb replay UI to have a more suggestive placeholder value and a submit button for accessibility:

<img width="1083" alt="screen shot 2018-07-03 at 9 36 01 pm" src="https://user-images.githubusercontent.com/2514780/42252173-4a6f2522-7f09-11e8-9472-b7441728ee4b.png">

...however, it is still unclear in this interface what it means to enter a URI and hit submit. What is it submitting and searching? What is "the archive"? Why am I entering a URI/URL and not free text, as is usually performed with large prominent text boxes?

Should we add a "help" or "more info"? Can we expect users to know what a "memento" is?
